Translations/Language.zh-cn.xml

```diff
@@ -1568,6 +1569,53 @@
+    <entry lang="zh-cn" key="RESCUE_DISK_CHECKLIST_B">确保应急文件存储在外部介质上。这可以是闪存驱动器、外部硬盘驱动器甚至是云存储。\n您的救援文件位于此处：</entry>
+    <entry lang="zh-cn" key="RESCUE_DISK_CHECKLIST_C">在开始加密系统之前，最好将个人数据备份到外部驱动器上，以防加密过程失败。</entry>
+    <entry lang="zh-cn" key="RESCUE_DISK_INFO_2">单击下面的下一步按钮以创建VeraCrypt 应急磁盘 ZIP 映像。</entry>
+    <entry lang="zh-cn" key="REMEMBER_PASSWORD"> 我记得我的密码</entry>
+    <entry lang="zh-cn" key="STORE_RESCUE_DISK"> 我存储了我的应急文件</entry>
+    <entry lang="zh-cn" key="BACKUP_DATA"> 我考虑过备份我的个人数据</entry>
+    <entry lang="zh-cn" key="CHECKLIST_TITLE">检查清单 - 在单击“下一步”之前</entry>
+    <entry lang="zh-cn" key="SYS_ENCRYPTION_PRETEST_INFO">现在必须进行一些测试以验证一切正常。\n\n对于此测试，您的计算机必须重新启动。然后您必须在 Windows 启动之前出现的 VeraCrypt 引导加载程序屏幕中输入您的密码。 Windows 启动后，VeraCrypt 会自动启动，您将被告知此预测试的结果。\n\n\n以下设备将被修改：驱动器 %c:</entry>
+    <entry lang="zh-cn" key="SYSENC_DECRYPTION_PAGE_INFO">请确保您的设备在加密过程中电量不会耗尽。\n如果解密被中断，您可以通过重新启动 VeraCrypt 并选择"系统">"继续被中断的进程"来恢复它。</entry>
+    <entry lang="zh-cn" key="ADVANCED_FEATURES">高级特性</entry>
+    <entry lang="zh-cn" key="VERY_WEAK_PASSWORD">非常弱</entry>
+    <entry lang="zh-cn" key="WEAK_PASSWORD">弱</entry>
+    <entry lang="zh-cn" key="MEDIUM_PASSWORD">中等</entry>
+    <entry lang="zh-cn" key="STRONG_PASSWORD">强</entry>
+    <entry lang="zh-cn" key="VERY_STRONG_PASSWORD">非常强</entry>
+    <entry lang="zh-cn" key="IDT_PASSWORD_STRENGTH">密码强度：</entry>
+    <entry lang="zh-cn" key="PASSWORD_HELP_SYSENC">您可以考虑将密码写在安全的地方。请注意，在您的密码达到所需的最低强度之前，确认密码的位置将显示为灰色。</entry>
+    <entry lang="zh-cn" key="AES_HELP_NEW">根据美国国家标准与技术研究院 (NIST) 和隐私与安全研究社区的说法，AES 是安全的。</entry>
+    <entry lang="zh-cn" key="SHA512_HELP">SHA-512 根据美国国家标准与技术研究院 (NIST) 和隐私与安全研究社区的要求是安全的。</entry>
+    <entry lang="zh-cn" key="CHOOSE_PASSWORD_TITLE">选择密码</entry>
+    <entry lang="zh-cn" key="TRANSLATED_PWD_NOT_WRITTEN">警告：写入转换后的密码时出错！请再试一次。如果此错误再次发生，请手动将键盘布局更改为美式键盘布局。</entry>
+    <entry lang="zh-cn" key="ERROR_TRANSLATE_PWD">警告：转换密码时出错！\n请再次输入密码并尝试。 如果此错误再次发生，请手动将键盘布局切换为美式键盘布局。</entry>
+    <entry lang="zh-cn" key="ERROR_KEYB_LAYOUT_NOT_LOADED">您的系统不支持标准的美式键盘布局。 请确保布局可用。 如需进一步帮助，请访问： https://support.microsoft.com/en-us/windows/manage-the-input-and-display-language-settings-in-windows-10-12a10cb4-8626-9b77-0ccb-5013e0c7c7a2 </entry>  
+    <entry lang="zh-cn" key="ERROR_PASTE_ACTION">粘贴文本被禁用。 请输入您的密码。</entry>
+    <entry lang="zh-cn" key="WARNING_PASSWORD_NOT_IDENTICAL">密码不一致！ 请在两个字段中输入相同的密码！</entry>
+    <entry lang="zh-cn" key="SYSENC_INFO">此助手将帮助您加密驱动器 (%c:)</entry>
+    <entry lang="zh-cn" key="SYSENC_INFO_2">可以通过物理访问您的设备的不良行为者很容易地绕过 Windows 密码。 驱动器加密有助于保护您的数据不被不良行为者访问。 VeraCrypt Drive Encryption 将通过使用附加密码保护您的数据来阻止它们。 </entry>
+    <entry lang="zh-cn" key="ADV_FEATURES_INTRO_TOOLTIP">在此处激活隐藏的操作系统。</entry>
+    <entry lang="zh-cn" key="ADV_FEATURES_NOTE">大多数专家同意以下预选选项是安全的。 仅当您知道自己在做什么时才更改它们。 </entry>
+    <entry lang="zh-cn" key="PIM_INFO">自定迭代次数 (PIM) 允许您设置密码哈希迭代的次数。 这为调整所需的安全级别提供了更大的灵活性，同时还控制了挂载/引导操作的性能。</entry>
```

Comment:
I can't find any 混杂 used in this language file. 

Meanwhile, for me alone, the more common statement I see is 哈希值 for hash value and 哈希算法/散列算法 for the hash algorithm. Since the statement 哈希 has been widely used in Simplified Chinese, I don't think it's a problem in the context, so I'll keep it as it is.

Reply:
>     <entry lang="zh-cn" key="IDC_LINK_HASH_INFO">混杂算法的更多信息（联网）</entry>
混杂 appeared in L48 of this file in master branch, L49 in this PR.
https://github.com/veracrypt/VeraCrypt/blob/master/Translations/Language.zh-cn.xml#L48

How about we use 哈希/哈希算法 for consistancy? I think it's better to not change it to something totally different as that might cause confusion to existing users.

Reply:
For L49, has been fixed in commit 28d53085e0169dca8fbc33dcc8596934bff7229d,
and for 哈希算法, I've checked every key contain "hash", and fixed them in 4d1b690247bc0579792b5668c3f7ab2e0ecc5d41.